language/German/strings.po

```diff
@@ -1188,8 +1164,7 @@ msgid ""
 msgstr ""
 "\n"
 "\n"
-"Drücken Sie OK oder bewegen Sie den Joystick nach links und dann nach rechts "
-"um fortzufahren"
+"Drücken Sie OK oder bewegen Sie den Joystick nach links und dann nach rechts, um fortzufahren"
```

Comment:
"Tippen Sie OK" would be a better translation. I will include this
suggestions in my PR later today.